### PR TITLE
Use Conda to do the crop mask

### PR DESCRIPTION
**Pull request by alexgleith**
* Because `rsgislib` is only available on Conda, install it from there.
* Rasterio is installed with `--no-binary`, because we have specific versions of stuff installed from Conda
* Pyproj is installed with Conda so we get a good version too.

### CODE REVIEW

File: production/cm_tools/setup.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
     "typing",
     "geopandas",
     "gdal",
-    "rsgislib",
+    # The absurdity of rsgislib is that it's installed and it's not able to be checked
+    # "rsgislib",
```
Comment (NikitaGandhi) [inline production/cm_tools/setup.py:27]: still it is project dependancy so don't you keep in requirement list?

Comment (alexgleith) [reply production/cm_tools/setup.py:27]: Thing is, you can't install it using Pip, so including it in here actually stops the build.

Comment (alexgleith) [reply production/cm_tools/setup.py:27]: Maybe it needs a better note.

**Review by NikitaGandhi — APPROVED** [review]
Great improvement!

**Comment by cbur24** [comment]
@alexgleith infinite gratitude!

**Review by cbur24 — APPROVED** [review]

**Pull request merged**